eng/installLatestFromReleaseBranch.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+# NuGet Feed URL
+nugetUrl="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json"
+
+# Package name to search for
+packageName="Microsoft.NET.Sdk.Aspire.Manifest-8.0.100"
+
+# Fetch the feed data
+feedData=$(curl -s $nugetUrl)
+
+# Extract resources and find the first package's metadata URL using jq
+metadataUrl=$(echo $feedData | jq -r '.resources[] | select(."@type" | contains("RegistrationsBaseUrl")) | ."@id"' | head -1)
+
+# Fetch the package data
+packageData=$(curl -s "${metadataUrl}${packageName}/index.json")
+
+# Get the latest version
+latestVersion=$(echo $packageData | jq -r '.items[-1].upper')
```

Comment:
I'm guessing they may not have jq. But perhaps they'll figure this out when it fails on this line..